tests/Unit/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProviderTest.php

```diff
@@ -607,6 +610,64 @@ private function getDatasetsForCategories(): array
         return $datasets;
     }
 
+    /**
+     * @return array
+     */
+    private function getDatasetsForPackedProducts(): array
+    {
+        $datasets = [];
+
+        $expectedOutputData = $this->getDefaultOutputData();
+        $productData = [
+            'packed_products' => [
+                0 => [
+                    'product_id' => 15,
+                    'productName' => 'wicked snowboard',
+                    'quantity' => 3,
+                    'reference' => 'demo_15',
+                    'combination_id' => null,
+                    'image' => null,
+                    'unique_identifier' => '15_0',
```

Comment:
You don't need this field to create a `PackedProductDetails` since it will be deduced from productId and combinationId

Reply:
you're right !